### DISCUSSION TITLE
MPPT max load current too high for the battery

### DISCUSSION THREAD

**Discussion by Kukusse** [hoylabs/OpenDTU-OnBattery#1085] [Q&A]
Hello,

I plan to implement the OpenDTU on Battery with a Victron MPPT 150/35 + 100/20, a pylonthec US2000 and a WR HM-600.
The pylontech has a maximal recommended charging load of 25A.

I calculated that, due to the different orientations of my pannels, the maximal current that the MPPT could produce is 38A.

Is there a way, using OpenDTU-on-Battery, to rule that if the total current produced by the MPPT is higher than a 25A, the difference is taken by the HM? It would be like a solar-pass-trough, but in the middle of a charging phase.

Exemple: On a sunny day, my 2 MPPTs produce together 33A. I want that only 25A goes to charge the battery, so I need to use 8A = 384W as limit to my Inverter.

I haven't seen this option in the documentation, but maybe it exists. As much as possible, I would like to avoid cutting the current on the MPPT.

The last option would be to invest more, in US3000, than the max current would ne no problem any more... I just need more euros...

Thanks for you answers.

Regards

**Comment by schlimmchen** [comment]
That's not supported. And I think it shouldn't be tried at all. OpenDTU-OnBattery is not a realtime control loop. Eventually, the sun will start shining, e.g., after a cloud passes, and the charge controllers will produce a large amount of power, such that the amperage goes up. OpenDTU-OnBattery will not be able to control the inverter(s) fast enough to prevent a current surge.

### The battery **must** be able to safely handle the maximum output current of the charge controllers.
